locales/en/index.yml

```diff
@@ -851,6 +851,16 @@ email:
     validation_ok: Your email has been validated. You can use all the functionalities of IO.
     validated: Email validated!
     validated_ok: Thank you, now you can use all payment features, messages forwarding, etc.
+  newvalidate:
```

Comment:
@AliceAzzolini the texts have changed and I have fixed the translations, could you check that they are ok?

Reply:
sure thing